scripts/binary/binary-cleanup.js

```diff
@@ -201,6 +201,8 @@ const buildEntryPointAndCleanup = async (buildAppDir) => {
     path.join(buildAppDir, '**', 'nexus', 'dist-esm'),
     path.join(buildAppDir, '**', '@graphql-tools', '**', '*.mjs'),
     path.join(buildAppDir, '**', 'graphql', '**', '*.mjs'),
+    path.join(buildAppDir, '**', '@openTelemetry', '**', 'esm'),
+    path.join(buildAppDir, '**', '@openTelemetry', '**', 'esnext'),
```

Comment:
these lines remove the esm and esnext folders from open telemetry, we don't use them.